.github/workflows/development-environment.yml

```
@@ -0,0 +1,20 @@
+name: bootstrap
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  dev-env-cli:
+    runs-on: macos-latest
+    timeout-minutes: 35
+    env:
+      STRAP_CI: 1
```

```suggestion
    env:
      STRAP_CI: 1
```

You could just use `CI` - it's supplied by GHA - https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

---

That's better. Thanks!